### ISSUE TITLE
Update mouse icon to look like most mice

### ISSUE THREAD

**Issue by philipzae** [elementary/icons#881]
## Prerequisites
- [X] I have searched open and closed issues for duplicates.

## Feature
**Is your feature request related to a problem? Please describe.**
Yes the mouse icon isn't easily understood at small sizes found in menus and doesn't like most mice used by the majority of people.

**Describe the solution you'd like**
Designing a new icon or utilizing one from another theme

**Existing work**
The humanity icon theme has a design that could be used.
![preview](https://imgur.com/FD3bN11.png)

**Describe alternatives you've considered**
Switching the icon theme to something else

**Additional context**
I've filed this bug downstream with Xubuntu's default icon theme as well.
https://github.com/shimmerproject/elementary-xfce/issues/214

**Comment by alfonsojon** [comment]
I believe the current mouse icon falsely implies likeness to the Apple Magic Mouse, which is an uncommon mouse that most users are not familiar with. I do think there should be a better mouse icon, one that is simple and easy for all users to understand without association to a certain brand.

**Comment by Fatih20** [comment]
I agree. Are you going to try to design one? If not, I'll be looking into it.

**Comment by philipzae** [comment]
> I agree. Are you going to try to design one? If not, I'll be looking into it.

Please go ahead and do so, while we await for the elementary team.

**Comment by marcosins** [comment]
(came here from the downstream issue)

I would like to add some references:

### Adwaita
16x16 | 48x48
------------ | -------------
![Mouse](https://raw.githubusercontent.com/GNOME/adwaita-icon-theme/mainline/Adwaita/16x16/legacy/input-mouse.png) | ![Mouse](https://raw.githubusercontent.com/GNOME/adwaita-icon-theme/mainline/Adwaita/48x48/legacy/input-mouse.png)
![Touchpad](https://raw.githubusercontent.com/GNOME/adwaita-icon-theme/mainline/Adwaita/16x16/legacy/input-touchpad.png) | ![Touchpad](https://raw.githubusercontent.com/GNOME/adwaita-icon-theme/mainline/Adwaita/48x48/legacy/input-touchpad.png)

### Yaru
16x16 | 48x48
------------ | -------------
![Mouse](https://raw.githubusercontent.com/ubuntu/yaru/master/icons/Suru/scalable/devices/input-mouse-symbolic.svg) | ![Mouse](https://raw.githubusercontent.com/ubuntu/yaru/master/icons/Suru/48x48/devices/input-mouse.png)
![Touchpad](https://raw.githubusercontent.com/ubuntu/yaru/master/icons/Suru/scalable/devices/input-touchpad-symbolic.svg) | **X**

**Comment by Fatih20** [comment]
I've designed a prototype, what do you think?
![image](https://user-images.githubusercontent.com/51940383/81461084-a9232080-91d3-11ea-811b-3c2c419615f4.png)

**Comment by philipzae** [comment]
@Fatih20 is that a top view of a mouse, as i'm not seeing it. Did you check the elementary HIG, as i'm not sure that the shadow at the bottom would be something that would be suitable.

**Comment by Fatih20** [comment]
> @Fatih20 is that a top view of a mouse, as i'm not seeing it. Did you check the elementary HIG, as i'm not sure that the shadow at the bottom would be something that would be suitable.

Yes it is. No, that's not the shadow, that's just part of the mouse.

**Comment by Fatih20** [comment]
Here is an alternative.
![image](https://user-images.githubusercontent.com/51940383/81461958-87c53300-91d9-11ea-94d0-07b225b70312.png)

**Comment by philipzae** [comment]
I'll await someone with knowledge of the HIG to suggest changes.

**Comment by danirabbit** [comment]
Fixed in #1310
